src/Tickets/Commerce/Order_Modifiers/Checkout/Abstract_Fees.php

```diff
@@ -319,13 +319,14 @@ protected function append_fees_to_cart( array $items, Value $subtotal ) {
 			// @todo - Review what needs to be sent. Some of these are used so wp_pluck doesn't cause a warning.
 			$items[] = [
 				'id'           => "fee_{$fee['id']}",
-				'price'        => $fee['fee_amount'],
-				'sub_total'    => $fee['fee_amount'],
+				'price'        => $fee['fee_amount']->get(),
+				'sub_total'    => $fee['fee_amount']->get(),
 				'type'         => 'fee',
 				'fee_id'       => $fee['id'],
 				'display_name' => $fee['display_name'],
 				'ticket_id'    => '0',
 				'event_id'     => '0',
+				'quantity'     => 1,
```

Comment:
I added quantity to fix an issue with the rest API erroring out while looking for the key.